src/ui/atoms/card/card.scss

```diff
@@ -18,39 +18,39 @@ $sizes-map: (
 );
 
 .okp4-card-main {
+  clip-path: polygon(30px 0, 100% 0, 100% 100%, 0 100%, 0 30px);
+  padding: 1px;
+  width: 100%;
+
   @include with-theme() {
-    clip-path: polygon(30px 0, 100% 0, 100% 100%, 0 100%, 0 30px);
-    padding: 1px;
+    @each $size in $sizes {
+      &.#{$size} {
+        max-width: map-get($sizes-map, #{$size}-width);
 
-    &.border {
-      background: themed(border);
-    }
+        &.border {
+          background: themed(border);
+        }
 
-    .okp4-card-container {
-      @each $color-key, $color-value in $colors {
-        display: flex;
-        flex-direction: column;
-        justify-content: space-between;
-        gap: 10px;
-        box-sizing: border-box;
-        clip-path: polygon(30px 0, 100% 0, 100% 100%, 0 100%, 0 30px);
-        max-height: map-get($sizes-map, large-height) - 2px;
-        max-width: map-get($sizes-map, large-width) - 2px;
-        overflow: auto;
-        word-break: break-word;
+        .okp4-card-container {
+          display: flex;
+          flex-direction: column;
+          justify-content: space-between;
+          gap: 10px;
+          box-sizing: border-box;
+          clip-path: polygon(30px 0, 100% 0, 100% 100%, 0 100%, 0 30px);
+          max-height: map-get($sizes-map, large-height);
+          overflow: auto;
+          word-break: break-word;
```

Comment:
`word-break` should not be carried by the parent element but rather by the child.

Reply:
100% right